### ISSUE TITLE
ImportError: No module named 'conda_api_q'

### ISSUE THREAD

**Issue by Nodd** [spyder-ide/spyder#2248]
## Description

* *What steps will reproduce the problem?*
1. Start spyder

* *What is the expected output? What do you see instead?*

I see the following traceback in the internal console:
```
ERROR: 3rd party plugin import failed for `p_condapackages`
Traceback (most recent call last):
  File "/usr/lib/python3.4/site-packages/spyderlib/otherplugins.py", line 53, in get_spyderplugins_mods
    __import__(name)
  File "/usr/lib/python3.4/site-packages/spyderplugins/p_condapackages.py", line 23, in <module>
    from spyderplugins.widgets.condapackagesgui import (CondaPackagesWidget,
  File "/usr/lib/python3.4/site-packages/spyderplugins/widgets/condapackagesgui.py", line 46, in <module>
    import conda_api_q
ImportError: No module named 'conda_api_q'
```

* *Please provide any additional information below*

I don't have Anaconda on this computer.

## Version and main components

* Spyder Version:  2.4.0dev
* Python Version:  3.4.3
* Qt Version    :  4.8.6, PyQt4 (API v2) 4.11.3 on Linux

## Optional dependencies

pyflakes >=0.6.0:  0.8.1 (OK)
pep8 >=0.6      :  1.6.2 (OK)
pygments >=1.6  :  2.0.2 (OK)
IPython >=1.0   :  3.0.0 (OK)
zmq >=2.1.11    :  14.5.0 (OK)
pandas >=0.13.1 :  0.15.2 (OK)
sphinx >=0.6.6  :  1.2.3 (OK)
psutil >=0.3    :  2.2.1 (OK)
rope >=0.9.2    :  0.9.4-1 (OK)
jedi >=0.8.1    :  0.8.1 (OK)
matplotlib >=1.0:  1.4.3 (OK)
sympy >=0.7.3   :  0.7.6 (OK)
pylint >=0.25   :  1.4.3 (OK)

**Comment by goanpeca** [comment]
hmm thats funny... will take a look

**Comment by goanpeca** [comment]
I rephrase... that is weird... `conda_api_q` is inside the `external-py3` and this is added to the path for python 3... so not sure what is going on

**Comment by Nodd** [comment]
Externals are not added any more, it messed things up with old versions of pep8 and path.py. ```conda_api_q``` should be somewhere else than externals I guess.

**Comment by goanpeca** [comment]
Well.... then that is why... can you try adding the file inside `conda_api_q` file inside external-py2 (or 3) into the widgets folder of the spyderplugins folder.... just to check?

**Comment by goanpeca** [comment]
Well I think now that conda was removed from the main repo, we can close this?

@Nodd ?

**Comment by ccordoba12** [comment]
Yes, let's close this one. Just for reference, let's mention here that this was fixed by commit c03f9293b1b79b1d28d30f4e8714401039b7e1c5
